Tests/PackageLoadingTests/PackageBuilderTests.swift

```diff
@@ -1832,7 +1833,7 @@ class PackageBuilderTests: XCTestCase {
             PackageBuilderTester(manifest, path: "/pkg", in: fs) { _, diagnostics in
                 diagnostics.check(diagnostic: "target path \'~/foo\' is not supported; it should be relative to package root", severity: .error)
             }
-        }
+        }*/
```

Comment:
@neonichu @abertelrud to confirm, per our discussion https://github.com/apple/swift-tools-support-core/pull/369#discussion_r1024661717 this is is correct to be removed

Reply:
That seems right to me.